tests/testthat/test-derive-array.R

```diff
@@ -185,7 +185,7 @@ with_test_authentication({
         ## No Data is added to the categories here
         expect_identical(
             names(categories(ds$metapetloc)),
-            c("Cat", "Dog", "Bird", "Skipped", "Not Asked", "one", "two", "No Data")
+            c("Cat", "Dog", "Bird", "Skipped", "Not Asked", "No Data", "one", "two")
```

Comment:
Is the change in the order because before "No Data" was being added, but only after the two arrays were merged?

Reply:
Correct.